test/index.js

```diff
@@ -483,6 +483,15 @@ describe('memoizeOne', () => {
     });
   });
 
+  describe('js typing', () => {
+    it('should maintain function arguments count', () => {
+      expect(memoizeOne(a => a).length).to.equal(1);
+      expect(memoizeOne((a, b) => a + b).length).to.equal(2);
+      expect(memoizeOne((...rest) => rest).length).to.equal(0);
```

Comment:
should this equal 0?
and should the next one equal 1?

Reply:
No, the final form of this function operates on `arguments`, not leaving anything in the function signature.
That's the problem.